recipe/meta.yaml

```diff
@@ -21,7 +21,8 @@ build:
 
 requirements:
     build:
-        - perl 5.26.0
+        - perl 5.26.0            #[not s390x]
+        - perl 5.26.2            #[s390x]
```

Comment:
Can we not have 5.26.* which will work for all the platforms?

Reply:
We can have, but intentionally not changed it because builds of other platforms are done already.

Reply:
ok